src/components/tasks/card/index.tsx

```diff
@@ -85,7 +85,7 @@ const Card: FC<Props> = ({
   
   function handleProgressColor(percentCompleted: number, startedOn: string, endsOn: string): string {
     const percentageofDays = getPercentageOfDays(startedOn, endsOn)
-    if(percentageofDays >= percentCompleted) {
+    if(percentCompleted === 100 && (percentageofDays >= percentCompleted || percentageofDays < 0)) {
```

Comment:
Nit: Also, for readability reasons, is `percentageofDays` the number of days left or the days that have passed? Could we update the variable name please?

`percentageofDaysLeft` or `percentageofDaysPassed`

Reply:
done